.gitattributes

```diff
@@ -1,2 +1,17 @@
 # Auto detect text files and perform LF normalization
 * text eol=lf
+
+/docs export-ignore
+/spec export-ignore
+/.editorconfig export-ignore
+/.eslintignore export-ignore
+/.eslintrc.json export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/appveyor.yml export-ignore
+/circle.yml export-ignore
+/package-lock.json export-ignore
+/CHANGELOG.md export-ignore
```

Comment:
I would personally like to keep CHANGELOG and README in the export because users locally see it in settings view

Reply:
I've pushed an update!